### PR TITLE
Abdulrahman

### PR DESCRIPTION
**Pull request by rahmandeprof**

### CODE REVIEW

File: abdulrahman/style.css [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+    height: 100%;
+    color: black;
+    position: relative;
+}
+
+.card__container {
+    display: flex;
+    width: 60%;
+    height: auto;
+    margin: 300px auto;
```
Comment (Jeremiahiro) [inline abdulrahman/style.css:21]: The width and margin of the auto will center your container thus you don't need the position and left attributes.
Also, when ever you have a margin above 100px, you MIGHT be doing the wrong thing. That margin is too much

File: abdulrahman/style.css [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+.card__logo {
+    height: 45px;
+    width: 40px;
+    margin-left: 15px;
+    margin-top: 15px;
+    border-radius: 50%;
+}
+
+.card__image {
+    height: 190px;
```
Comment (Jeremiahiro) [inline abdulrahman/style.css:43]: You height and width does not align and this makes the image looks skewed

**Review by Jeremiahiro — CHANGES_REQUESTED** [review]
Nice attempt, fix the requested changes

File: abdulrahman/style.css (hunk 2 above)
Comment (rahmandeprof) [reply abdulrahman/style.css:43]: notes taken. will make sure to perfect everything just very soon

**Pull request closed without merging**